site/src/components/Footer/Footer.tsx

```diff
@@ -55,12 +56,12 @@ export const Footer: React.FC<React.PropsWithChildren<FooterProps>> = ({ buildIn
 
 const useFooterStyles = makeStyles((theme) => ({
   root: {
-    opacity: 0.6,
+    color: colors.gray[7],
```

Comment:
I tried the `palette.text.secondary` but for this background and how "soft" we want the color I decided to use the color directly since it is very specific to this component.